### PR TITLE
minor code fixes

### PR DESCRIPTION
**Pull request by rschalch**
Adjusting some minor issues :)

### CODE REVIEW

File: aiohttp/helpers.py [hunk 1]
```diff
@@ -43,7 +43,7 @@
 
 class _BaseCoroMixin(Coroutine):
 
-    __slots__ = ('_coro')
+    __slots__ = '_coro'
```
Comment (hellysmile) [inline aiohttp/helpers.py]: maybe here is typo? it should be tuple, as for me comma was missed

File: aiohttp/test_utils.py [hunk 2]
```diff
@@ -500,7 +500,7 @@ def make_mocked_request(method, path, headers=None, *,
     if headers:
         headers = CIMultiDict(headers)
         raw_hdrs = tuple(
-            (k.encode('utf-8'), v.encode('utf-8')) for k, v in headers.items())
+            (k.encode(), v.encode('utf-8')) for k, v in headers.items())
```
Comment (hellysmile) [inline aiohttp/test_utils.py]: I guess here 'utf-8' haven't been removed

File: aiohttp/client_proto.py [hunk 3]
```diff
@@ -10,7 +10,7 @@
 class ResponseHandler(DataQueue, asyncio.streams.FlowControlMixin):
     """Helper class to adapt between Protocol and StreamReader."""
 
-    def __init__(self, *, loop=None, **kwargs):
```
Comment (asvetlov) [inline aiohttp/client_proto.py:13]: Why?

File: aiohttp/client_proto.py [hunk 4]
```diff
@@ -70,7 +70,7 @@ def connection_lost(self, exc):
 
         try:
             uncompleted = self._parser.feed_eof()
-        except Exception as e:
```
Comment (asvetlov) [inline aiohttp/client_proto.py:73]: `e` is never used.

File: aiohttp/http_websocket.py [hunk 5]
```diff
@@ -353,7 +353,7 @@ def _feed_data(self, data):
 
                     if opcode == WSMsgType.TEXT:
                         try:
-                            text = payload_merged.decode('utf-8')
```
Comment (asvetlov) [inline aiohttp/http_websocket.py]: I prefer to point encoding explicitly

File: aiohttp/streams.py [hunk 6]
```diff
@@ -451,14 +451,14 @@ def feed_data(self, data):
     async def readline(self):
         return b''
 
-    async def read(self, n=-1):
```
Comment (asvetlov) [inline aiohttp/streams.py]: Signature change is backward incompatible.

File: aiohttp/web_request.py [hunk 7]
```diff
@@ -589,7 +589,6 @@ def __repr__(self):
     @asyncio.coroutine
     def _prepare_hook(self, response):
         return
-        yield  # pragma: no cover
```
Comment (asvetlov) [inline aiohttp/web_request.py]: The line is intentional

File: aiohttp/web_ws.py [hunk 8]
```diff
@@ -282,7 +282,7 @@ def send_json(self, data, *, dumps=json.dumps):
                     waiter = self._waiting
                     self._waiting = None
                     waiter.set_result(True)
-            except (asyncio.CancelledError, asyncio.TimeoutError) as exc:
```
Comment (asvetlov) [inline aiohttp/web_ws.py:284]: `exc` is not used

File: aiohttp/web_urldispatcher.py [hunk 9]
```diff
@@ -349,7 +349,7 @@ def __repr__(self):
 
 class DynamicResource(Resource):
 
-    DYN = re.compile(r'\{(?P<var>[_a-zA-Z][_a-zA-Z0-9]*)\}')
+    DYN = re.compile(r'{(?P<var>[_a-zA-Z][_a-zA-Z0-9]*)\}')
     DYN_WITH_RE = re.compile(
         r'\{(?P<var>[_a-zA-Z][_a-zA-Z0-9]*):(?P<re>.+)\}')
```
Comment (kxepal) [inline aiohttp/web_urldispatcher.py]: Looks like not all curve brackets were unescaped.

File: aiohttp/client_proto.py (hunk 3 above)
Comment (rschalch) [reply aiohttp/client_proto.py:13]: kwargs were not being used at this `__init__`

File: aiohttp/client_proto.py (hunk 4 above)
Comment (rschalch) [reply aiohttp/client_proto.py:73]: That's why I removed it

File: aiohttp/web_ws.py (hunk 8 above)
Comment (rschalch) [reply aiohttp/web_ws.py:284]: That's why I removed it

File: aiohttp/test_utils.py [hunk 10]
```diff
@@ -33,7 +33,7 @@ def run_briefly(loop):
 
 def unused_port():
     """Return a port that is unused on the current host."""
-    with socket.socket(socket.AF_INET, socket.SOCK_STREAM) as s:
```
Comment (asvetlov) [inline aiohttp/test_utils.py]: Please keep explicit family and protocol

File: aiohttp/test_utils.py [hunk 11]
```diff
@@ -55,7 +55,7 @@ def __init__(self, *, scheme=sentinel, loop=None,
         if self.server:
             return
         self._loop = loop
-        self._socket = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+        self._socket = socket.socket()
```
Comment (asvetlov) [inline aiohttp/test_utils.py]: Please save explicit family and protocol

File: aiohttp/web_urldispatcher.py [hunk 12]
```diff
@@ -349,9 +349,9 @@ def __repr__(self):
 
 class DynamicResource(Resource):
 
-    DYN = re.compile(r'\{(?P<var>[_a-zA-Z][_a-zA-Z0-9]*)\}')
+    DYN = re.compile(r'{(?P<var>[_a-zA-Z][_a-zA-Z0-9]*)\}')
```
Comment (webknjaz) [inline aiohttp/web_urldispatcher.py]: What about closing curly brace `\}` here

File: aiohttp/web_urldispatcher.py [hunk 13]
```diff
@@ -349,9 +349,9 @@ def __repr__(self):
...
     DYN_WITH_RE = re.compile(
-        r'\{(?P<var>[_a-zA-Z][_a-zA-Z0-9]*):(?P<re>.+)\}')
+        r'{(?P<var>[_a-zA-Z][_a-zA-Z0-9]*):(?P<re>.+)\}')
```
Comment (webknjaz) [inline aiohttp/web_urldispatcher.py]: and here.

File: aiohttp/web_urldispatcher.py [hunk 14]
```diff
@@ -573,7 +573,7 @@ def __iter__(self):
         return ret
 
     def _directory_as_html(self, filepath):
-        "returns directory's index as html"
+        """returns directory's index as html"""
```
Comment (webknjaz) [inline aiohttp/web_urldispatcher.py]: Since you already touch docstring, please follow PEP257: `"""Render directory index as html."""`

**Comment by asvetlov** [comment]
Please resolve conflicts.

File: aiohttp/web_urldispatcher.py [hunk 15]
```diff
@@ -323,9 +323,9 @@ def __repr__(self):
 
 class DynamicResource(Resource):
 
-    DYN = re.compile(r'\{(?P<var>[_a-zA-Z][_a-zA-Z0-9]*)\}')
+    DYN = re.compile(r'{(?P<var>[_a-zA-Z][_a-zA-Z0-9]*)}')
     DYN_WITH_RE = re.compile(
-        r'\{(?P<var>[_a-zA-Z][_a-zA-Z0-9]*):(?P<re>.+)\}')
+        r'{(?P<var>[_a-zA-Z][_a-zA-Z0-9]*):(?P<re>.+)}')
```
Comment (asvetlov) [inline aiohttp/web_urldispatcher.py:328]: Looks like `{}` should be escaped: https://bugs.python.org/issue32067

**Comment by asvetlov** [comment]
Implemented the most of proposed fixes in fab06f2d
@rschalch thanks!

**Pull request closed without merging**